packs/data/other-effects.db/effect-2-circumstance-penalty-to-ranged-attacks.json

```
@@ -0,0 +1,50 @@
+{
+    "_id": "wVGSlZEGxsg1s8AZ",
+    "img": "systems/pf2e/icons/effects/crtical-hit-fumble.webp",
+    "name": "Effect: -2 circumstance penalty to ranged attacks",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by Critical Hit/Fumble Deck<br /></strong>-2 circumstance penalty to ranged attacks</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to ranged attacks",
```

```suggestion
                "label": "circumstance penalty to ranged attacks",
```
We should localize these, and maybe just have a generic label for the critical hit and critical failure deck rather than explode re-en.json with them.

---

Yea, I wasn't sure about exploding out the re-en.json until reviewed.  Originally I wanted descriptive text in the Effects Panel so you could remember (especially as a GM) what the "effect" was for but needed shorter labels in chat messages.

I'll do a brush up and see how it looks with a more generic/localized label that can be reused...